saskatoon/sitebase/templates/registration/change_password.html

```diff
@@ -0,0 +1,65 @@
+{% extends 'app/base/view_empty.html' %}
+{% load i18n %}
+{% load static %}
+{% block body %}
+    <style>
+      li::before {
+        content: "• ";
+      }
+    </style>
+    <div class="login-content">
+        <!-- Login -->
+        <div class="nk-block toggled" id="l-login">
+            <form action='{% url "change_password" %}' method="post">
+                {% csrf_token %}
+                <div class="nk-form">
+                    <h1>Change password</h1>
+                    {% block form %}
+                        {% if form.errors %}
+                            <div class="alert-list">
+                                <div class="alert alert-danger alert-mg-b-0"
+                                     role="alert"
+                                     style="text-align: left;
+                                            display: flex;
+                                            flex-flow: column;
+                                            gap: 1em">
+                                    <!-- Django renders errors with the attr keys as text inside ul component. -->
+                                    <!-- This is a workaround to avoid writing a new template for error rendering. -->
+                                    {% if form.errors.old_password %}<div>{{ form.errors.old_password }}</div>{% endif %}
+                                    {% if form.errors.new_password1 %}<div>{{ form.errors.new_password1 }}</div>{% endif %}
+                                    {% if form.errors.new_password2 %}<div>{{ form.errors.new_password2 }}</div>{% endif %}
+                                </div>
+                                <br>
+                            </div>
+                        {% endif %}
```

Comment:
have you tried using simply `{{form |crispy}}` ?
see example: https://github.com/LesFruitsDefendus/saskatoon-ng/blob/9eee8ffba13f0978916fab800ec3970be38a8737/saskatoon/sitebase/templates/app/forms/model_form.html#L6

Reply:
not sure how `crispy works, but doesn't seem to work with errors.

errors render as unordered lists `<ul>...<li>`

Reply:
switched to crispy and added basic styling to match
![20240503_17h42m21s_grim](https://github.com/LesFruitsDefendus/saskatoon-ng/assets/75142042/5504afc5-a265-48da-b063-92a039835850)